crates/mehen-sql/tests/fixtures/bigquery_scripting.sql

```
@@ -0,0 +1,27 @@
+-- sqlfluff:dialect:bigquery
+declare x int64 default 0;
+
+if x > 0 then
+  update t set c = 1 where id = x;
```

SQLFluff (4.2.2)

[error] The 'where' keyword should always start a new line.

(LT14)